docs/submitting.md

```diff
@@ -213,26 +219,27 @@ In addition, the software associated with your submission must:
 
 ## Authorship
 
 Purely financial (such as being named on an award) and organizational (such as general supervision of a research group) contributions are not considered sufficient for co-authorship of JOSS submissions, but active project direction and other forms of non-code contributions are. The authors themselves assume responsibility for deciding who should be credited with co-authorship, and co-authors must always agree to be listed. In addition, co-authors agree to be accountable for all aspects of the work, and to notify JOSS if any retraction or correction of mistakes are needed after publication.
 
 ## Submissions using proprietary languages/dev environments
 
-We strongly prefer software that doesn't rely upon proprietary (paid for) development environments/programming languages. However, provided _your submission_ meets our submission requirements (including having a valid open source license) then we will consider your submission for review. Should your submission be accepted for review, we may ask you, the submitting author, to help us find reviewers who already have the required development environment installed.
+We strongly prefer software that doesn't rely upon proprietary (paid for) development environments/programming languages. However, provided _your software_ meets our submission requirements (including having a valid open source license) then we will consider your submission for review. Should your submission be accepted for review, we may ask you, the submitting author, to help us find reviewers who already have the required development environment installed.
 
 ## The review process
 
 After submission:
 
-- One or more JOSS reviewers are assigned and the review is carried out in the [reviews repository](https://github.com/openjournals/joss-reviews).
-- Authors respond to reviewer-raised issues (if any are raised) on the submitted repository's issue tracker. Reviewer contributions, like any other contributions, should be acknowledged in the repository.
-- Upon successful completion of the review, deposit a copy of your (updated) repository with a data-archiving service such as [Zenodo](https://zenodo.org/) or [figshare](https://figshare.com/), issue a DOI for the archive, and update the review issue thread with your DOI.
-- After assignment of a DOI, your paper metadata is deposited in CrossRef and listed on the JOSS website.
-- And that's it.
+- An Associate Editor-in-Chief will carry out an initial check of your submission, and proceed to assign a handling editor.
+- The handling editor will assign two or more JOSS reviewers, and the review will be carried out in the [JOSS reviews repository](https://github.com/openjournals/joss-reviews).
+- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer contributions, like any other contributions, should be acknowledged in the repository.
```

Comment:
Should we say "reviewer and editor contributions, ..."?

Reply:
That's old text that I did not touch. Unclear why it appears green in the diff.

Reply:
I think just because it moved.  But my question still stands; if we are making changes, should we change this too?

Reply:
I would leave as is. Item is about reviewer actions.

Reply:
I think the section is about what the author should expect after submission, but I see what you mean in this specific item. I don't have a good suggestion - maybe someone else will.

Reply:
I kind of agree with making it "reviewer and editor contributions..." (perhaps even more so for the "reviewer-raised issues") since this document is for the submitter, for whom it doesn't really matter if it's the editor or reviewer that comments, and in rare cases the editor could also be a reviewer.